### PR TITLE
New Default Setting

### PR DESCRIPTION
**Pull request by chsprenger**

### CODE REVIEW

**Comment by chsprenger** [comment]
Update treatment steps

**Comment by mrustl** [comment]
@chsprenger you have only pushed your changes we made together yesterday! Maybe there are still some changes still on your laptop?

**Comment by mrustl** [comment]
Now I am happy! @Roschl have I look how @chsprenger performed!

**Comment by mrustl** [comment]
@chsprenger you also need to update the https://github.com/inowas/frontend-app/blob/dev/src/scenes/t15/components/defaults/processes.ts in order to be in line with `removals.ts`

**Comment by chsprenger** [comment]
Si claro, is´ in Mache.

Von: Michael Rustler ***@***.***
Gesendet: Montag, 14. Juni 2021 20:52
An: inowas/frontend-app ***@***.***>
Cc: Christoph Sprenger ***@***.***>; Mention ***@***.***>
Betreff: Re: [inowas/frontend-app] New Default Setting (#397)


@chsprenger<https://github.com/chsprenger> you also need to update the https://github.com/inowas/frontend-app/blob/dev/src/scenes/t15/components/defaults/processes.ts in order to be in line with removals.ts

File: src/scenes/t15/components/defaults/processes.ts [hunk 1]
```diff
@@ -82,14 +82,19 @@ const processes: IProcessDefault[] = [
   },
   {
     'TreatmentName': 'Primary treatment',
```
Comment (Roschl) [inline src/scenes/t15/components/defaults/processes.ts]: Did you mix up name and group here? In the other file "Quiescent basin" is the name and not the group.

File: src/scenes/t15/components/defaults/processes.ts [hunk 2]
```diff
@@ -82,14 +82,19 @@ const processes: IProcessDefault[] = [
...
-    'TreatmentGroup': 'Pretreatment',
+    'TreatmentGroup': 'Quiescent basin',
     'TreatmentDescription': 'Primary treatment consists of temporarily holding the sewage in a quiescent basin where heavy solids can settle to the bottom while oil, grease and lighter solids float to the surface. The settled and floating materials are removed and the remaining liquid may be discharged or subjected to secondary treatment'
   },
   {
     'TreatmentName': 'Secondary treatment',
-    'TreatmentGroup': 'Pretreatment',
+    'TreatmentGroup': 'Activated sludge',
```
Comment (Roschl) [inline src/scenes/t15/components/defaults/processes.ts]: Same here? Isn't "Activated sludge" the name and not the group?

**Review by Roschl — CHANGES_REQUESTED** [review]
Please check name vs group (see my comments in the code)

**Review by Roschl — APPROVED** [review]

**Pull request merged**